.circleci/config.yml

```diff
@@ -52,11 +52,11 @@ references:
       key: *nightly_cache_key
       paths:
         - ~/.stack
-        - ~/.stack-work
+        - .stack-work
```

Comment:
We need to do two things:

1. Update ALL cache key prefixes `s/v1-/v2-/g` <- that's the only way to invalidate the cache.
2. Ensure that the _relative_ path is cached properly. I think it needs to be an absolute path.

Reply:
Absolute/relative doesn't matter as per their documentation: https://circleci.com/docs/2.0/caching/

Reply:
And can verify from work (we use circle 2.0) that caching local directories work

Reply:
awesome, thank you